service/src/main/java/uk/gov/hmcts/reform/fpl/service/ChildrenMigrationService.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.enums.PartyType;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.Child;
+import uk.gov.hmcts.reform.fpl.model.ChildParty;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+import static java.util.stream.Collectors.toList;
+
+@Service
+public class ChildrenMigrationService {
+
+    @Autowired
+    private ObjectMapper mapper;
+
+    public String setMigratedValue(CaseData caseData) {
+        if (caseData.getChildren1() != null || caseData.getChildren() == null) {
+            return "Yes";
+        } else {
+            return "No";
+        }
+    }
+
+    @SuppressWarnings("unchecked")
+    public List<Element<Child>> expandChildrenCollection(CaseData caseData) {
+        if (caseData.getChildren1() == null) {
+            List<Element<Child>> populatedChildren = new ArrayList<>();
+
+            populatedChildren.add(Element.<Child>builder()
+                .value(Child.builder()
+                    .party(ChildParty.builder()
+                        .partyId(UUID.randomUUID().toString())
+                        .build())
+                    .build())
+                .build());
+            return populatedChildren;
+        } else {
+            return caseData.getChildren1();
+        }
+    }
+
+    @SuppressWarnings("unchecked")
```

Comment:
not needed anymore please remove

Reply:
Not removed yet so I will unresolved the conversation